recipes/pipewire/all/conanfile.py

```diff
@@ -0,0 +1,439 @@
+        tc.project_options["sdl2"] = "disabled"
+        tc.project_options["selinux"] = feature("with_selinux")
+        tc.project_options["session-managers"] = []
+        tc.project_options["sndfile"] = feature("with_libsndfile")
+        tc.project_options["spa-plugins"] = "enabled"
+        tc.project_options["sysconfdir"] = os.path.join(self.package_folder, "res")
+        tc.project_options["systemd"] = "disabled"
+        tc.project_options["systemd-user-service"] = "disabled"
+        tc.project_options["tests"] = "disabled"
+        tc.project_options["udev"] = feature("with_libudev", True)
+        tc.project_options["udevrulesdir"] = os.path.join(
+            self.package_folder, "res", "udev", "rules.d"
+        )
+        tc.project_options["vulkan"] = feature("with_vulkan")
+        tc.project_options["x11"] = feature("with_x11")
+        tc.project_options["x11-xfixes"] = feature("with_xfixes")
+        tc.generate()
+
+    def _patch_sources(self):
+        apply_conandata_patches(self)
+
+    def build(self):
+        self._patch_sources()
+        meson = Meson(self)
+        meson.configure()
+        meson.build()
+
+    @property
+    def _libpipewire_api_version_txt(self):
+        return os.path.join(self.package_folder, "res", "libpipewire-api-version.txt")
+
+    @property
+    def _libspa_api_version_txt(self):
+        return os.path.join(self.package_folder, "res", "libspa-api-version.txt")
+
+    def package(self):
+        copy(
+            self,
+            "COPYING",
+            self.source_folder,
+            os.path.join(self.package_folder, "licenses"),
+        )
+        meson = Meson(self)
+        meson.install()
+
+        libpipewire_pc_file = glob.glob(
+            os.path.join(self.package_folder, "lib", "pkgconfig", "libpipewire-*.pc")
+        )[0]
+        libspa_pc_file = glob.glob(
+            os.path.join(self.package_folder, "lib", "pkgconfig", "libspa-*.pc")
+        )[0]
+
+        libpipewire_api_version = os.path.splitext(
+            os.path.basename(libpipewire_pc_file)
+        )[0].split("-")[1]
+        libspa_api_version = os.path.splitext(os.path.basename(libspa_pc_file))[
+            0
+        ].split("-")[1]
+
+        save(self, self._libpipewire_api_version_txt, libpipewire_api_version)
+        save(self, self._libspa_api_version_txt, libspa_api_version)
+
+        rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
+        rmdir(self, os.path.join(self.package_folder, "share"))
+        rm(self, "*.pdb", os.path.join(self.package_folder, "lib"))
+        rm(self, "*.pdb", os.path.join(self.package_folder, "bin"))
+
+        fix_apple_shared_install_name(self)
+
+    def package_info(self):
+        libpipewire_api_version = load(self, self._libpipewire_api_version_txt).strip()
+        libspa_api_version = load(self, self._libspa_api_version_txt).strip()
+
+        self.runenv_info.prepend_path(
```

Comment:
Are you sure that prepending/appending is correct here? I.e. would the library be able to use more than one path in the envvars?

Reply:
Looking at what's documented, I bet you're on the right track there. I'll change this to define it.

Reply:
Done.